met-web/src/apiManager/endpoints/index.ts

```diff
@@ -24,7 +24,7 @@ const Endpoints = {
         CREATE: `${AppConfig.apiUrl}/submission/`,
     },
     Comment: {
-        GET_ALL: `${AppConfig.apiUrl}/comments/`,
+        GET_ALL: `${AppConfig.apiUrl}/comment/all/survey/survey_id`,
```

Comment:
I think this is more likely /survey/<survey_id>/comments/ but we have a task to refactor the endpoints so no worries for now

Reply:
nope, lets use the task to refactor it

Reply:
since we have time now in this sprint , i would vote for doing it now..
All is not much used in the rest..
as @tiagograf-AOT said /survey/<survey_id>/comments is the correct one here..